src/test/java/de/uhd/ifi/se/decision/management/jira/consistency/TestCipCalculation.java

```diff
@@ -15,34 +15,66 @@
 import java.util.stream.Collectors;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertNull;
 
 public class TestCipCalculation extends TestSetUp {
 
-	private static Issue baseIssue;
+	private static List<MutableIssue> testIssues;
 
 	@BeforeClass
 	public static void setUp() {
+		TestSetUp.init();
 		Project project = JiraProjects.getTestProject();
-		List<MutableIssue> testIssues = JiraIssues.createJiraIssues(project);
-		TestCipCalculation.baseIssue = testIssues.get(0);
+		TestCipCalculation.testIssues = JiraIssues.createJiraIssues(project);
 	}
 
 	@Test
 	public void testCIP() {
-		ContextInformation contextInformation = new ContextInformation(TestCipCalculation.baseIssue);
+		Issue baseIssue = TestCipCalculation.testIssues.get(0);
+		ContextInformation contextInformation = new ContextInformation(baseIssue);
 		try {
 			Collection<LinkSuggestion> linkSuggestions = contextInformation.getLinkSuggestions();
 			List<LinkSuggestion> sortedLinkSuggestions = linkSuggestions
 				.stream()
 				.sorted((LinkSuggestion::compareTo))
 				.collect(Collectors.toList());
+			LinkSuggestion identicalIssueSuggestion = sortedLinkSuggestions.get(sortedLinkSuggestions.size() - 1);
 			assertEquals("The baseIssue should be the most similar to itself.", baseIssue.getKey(),
-				sortedLinkSuggestions.get(sortedLinkSuggestions.size() - 1).getTargetIssue().getKey());
+				identicalIssueSuggestion.getTargetIssue().getKey());
+			assertNotNull(identicalIssueSuggestion.getScore().getScores());
+
+			assertEquals("The baseIssue should be set correctly.", baseIssue.getKey(),
+				identicalIssueSuggestion.getBaseIssue().getKey());
+
 		} catch (NullPointerException | GenericEntityException e) {
-			System.out.println("ERROR:");
+			System.err.println("ERROR:");
 			e.printStackTrace();
 			assertNull(e);
 		}
 	}
+
+	@Test
+	public void testTracingCIP() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5586048)